CHANGELOG.md

```diff
@@ -1,22 +1,44 @@
 # Cromwell Change Log
 
-## 0.20
+## 0.22
```

Comment:
ToL: I thought we added a git flag along the lines of "purely additive" so that the diffs wouldn't get confused by simple additions?